### PR TITLE
trying to update the .md for VSMac instructions from a blank project

### PR DESCRIPTION
**Pull request by jamessdixon**

### CODE REVIEW

File: docs/index.md [hunk 1]
```diff
@@ -37,6 +37,16 @@ By default iOS and Android projects are created. For WPF apps use `--WPF`, for e
    Android only: dotnet new fabulous-app -n SqueakyApp --iOS=false
    iOS only:     dotnet new fabulous-app -n SqueakyApp --Android=false
    WPF only:     dotnet new fabulous-app -n SqueakyApp --WPF --Android=false --iOS=false
+   
+5. If you are using Visual Studio for Mac and you want to start with File -> New make sure you target ".NE Standard" to add the references to Fabulous:
```
Comment (TimLariviere) [inline docs/index.md]: Small typo: ".NE Standard" => ".NET Standard"
Could you also add a comma between `File -> New` and `make sure ...` to make it easier to read?

File: docs/index.md [hunk 2]
```diff
@@ -37,6 +37,16 @@ By default iOS and Android projects are created. For WPF apps use `--WPF`, for e
...
+       
+       File -> New Solution
+       
+       Multiplatform App -> Blank Forms App (F#)
+       
+       Shared Code -> Use .NET Standard
+       
+       <img src="https://user-images.githubusercontent.com/7204669/9c43285c-bfcf-44ef-b0f5-e912f179e406.png" width="100">
```
Comment (TimLariviere) [inline docs/index.md]: I'm having a `Access Denied` on the linked image
Also I think the tabulation on this line makes it part of a quotation, disabling images.

Comment (TimLariviere) [reply docs/index.md]: @jamessdixon If you have some time, could you see this comment?
We can merge the PR once we have a good image link.

**Comment by jamessdixon** [comment]
@TimLariviere how can I upload an image to GitHub?  How did you do it for the ogirinal?  Is there a doc somewhere?

**Comment by TimLariviere** [comment]
@jamessdixon I think drag & drop on GitHub site was used for the other images.
GitHub automatically uploads and gives you a public url when doing this.

If I'm not mistaking, the image you put in your other PR is the one you want to show here?
https://user-images.githubusercontent.com/1394644/45930814-97bfce80-bf32-11e8-8f7a-ebcbcb0247fa.png

**Comment by TimLariviere** [comment]
@jamessdixon I think drag & drop on GitHub site was used for the other images.
GitHub automatically uploads and gives you a public url when doing this.

If I'm not mistaking, the image you put in your other PR is the one you want to show here?
https://user-images.githubusercontent.com/1394644/45930814-97bfce80-bf32-11e8-8f7a-ebcbcb0247fa.png

**Comment by TimLariviere** [comment]
@jamessdixon I think drag & drop on GitHub site was used for the other images.
GitHub automatically uploads and gives you a public url when doing this.

If I'm not mistaking, the image you put in your other PR is the one you want to show here?
https://user-images.githubusercontent.com/1394644/45930814-97bfce80-bf32-11e8-8f7a-ebcbcb0247fa.png

**Comment by TimLariviere** [comment]
@jamessdixon I think drag & drop on GitHub site was used for the other images.
GitHub automatically uploads and gives you a public url when doing this.

If I'm not mistaking, the image you put in your other PR is the one you want to show here?
https://user-images.githubusercontent.com/1394644/45930814-97bfce80-bf32-11e8-8f7a-ebcbcb0247fa.png

**Comment by jamessdixon** [comment]
ok, I updated the uri. Thanks

**Comment by TimLariviere** [comment]
Thanks!

**Pull request merged**